### DISCUSSION TITLE
When tmux will be updated to 3.4?

### DISCUSSION THREAD

**Discussion by orzomaxx** [termux/termux-packages#19729] [Q&A] [answered]
Tmux version `3.4` was released 13 February 2024

Termux repository contains tmux version `3.3a`, that somehow reports itself (via `tmux -V`) as `next-3.4` version (which is strange and misleading)

I don't know where to ask about it, so I ask it here: When tmux will be updated in Termux repository?

**Comment by twaik** [comment, marked as answer]
No need to ask for package updates. We see all new versions as soon as they are available but have no enough people to update packages. Pull requests are always welcome.

**Comment by TomJo2000** [comment]
Got it building fine, will send in a PR shortly.
I'd appreciate if you could test the package before merging.

**Reply by TomJo2000** [reply]
#19736
Testing would be much appreciated.
